src/library/scala/collection/mutable/ArrayDeque.scala

```diff
@@ -253,7 +253,7 @@ class ArrayDeque[A] protected (
     elem
   }
 
-  def subtractOne(elem: A): this.type = {
+  override def subtractOne(elem: A): this.type = {
     val idx = indexOf(elem)
     if (idx >= 0) remove(idx, 1) //TODO: SeqOps should be fluent API
     this
```

Comment:
Then I think we can remove this override since it does the same as the base implementation.

Reply:
As discussed, I'm leaving it as it is, because `remove(idx)` and `remove(idx,1)` aren't exactly the same (the former gets the element doing bound checks before calling `remove(idx,1)` ) It might not matter, but  just in case.